fenix/app/metrics.yaml

```diff
@@ -7754,6 +7754,53 @@ addons:
     metadata:
       tags:
         - WebExtensions
+  dialog_try_restart_process_clicked:
+    type: counter
+    lifetime: application
+    description: |
+      A counter that indicates the number of times that a user
+      has clicked on the button try to restart add-ons
+      on the dialog for when the extensions process crashed.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1850350
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1850350#c2
+      - https://github.com/mozilla-mobile/firefox-android/pull/3472
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: never
+    metadata:
+      tags:
+        - WebExtensions
+  dialog_disable_process_clicked:
+    type: counter
+    lifetime: application
+    description: |
+      A counter that indicates the number of times that a user
+      has clicked on the button continue without add-ons
```

Comment:
with* add-ons (not without)

Reply:
Checked the strings on Fenix, you are the screenshot on the PR maybe outdate 😅 
![image](https://github.com/mozilla-mobile/firefox-android/assets/773158/fb9350a8-1b4f-4148-8721-eb02b566fd10)

Reply:
it's either "continue without addons" OR "continue with add-ons disabled"